.github/workflows/check-prs.yml

```
@@ -0,0 +1,64 @@
+    permissions:
+      contents: read
+      pull-requests: write
+      issues: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Check for inactive PRs
+        uses: actions/github-script@v7
+        with:
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 27: did not find expected key

(syntax-check)